src/components/Minting/Minting.tsx

```diff
@@ -107,34 +125,51 @@ const Minting: FC = () => {
 				toast.error(MINT_FAIL_GENERIC)
 			}
 			console.error(err.error)
+		} finally {
+			setTxPending(false)
 		}
 	}
 
 	return (
 		<div className={classes.page}>
-			<h1>{MINT_PAGE_TITLE}</h1>
 			{loading ? (
 				<Spinner />
-			) : (contractState === 0 ? (
+			) : totalSupply || 0 >= MAX_SUPPLY ? (
 				<>
-					<p>Sale not yet live!</p>
-					<p>Come back soon!</p>
+					<h1>{MINT_SOLDOUT_TITLE}</h1>
+					<p>{MINT_SOLDOUT_SUBTEXT}</p>
+					<a href={OPENSEA_LINK} target="_blank">
+						<OpenseaLogo />
+					</a>
+				</>
+			) : contractState === 0 ? (
+				<>
+					<h1>{MINT_NOSALE_TITLE}</h1>
+					<p>{MINT_NOSALE_SUBTEXT}</p>
 				</>
 			) : (
-				<div className={classes.mint}>
-					<Input
-						id="qty"
-						type="number"
-						defaultValue={allowance}
-						min={1}
-						max={allowance}
-						required
-					/>
-					<Button onClick={doMint} disabled={txPending}>
-						{txPending ? TX_PENDING : MINT_BTN_TEXT}
-					</Button>
-				</div>
-			))}
+				<>
+					<h1>{MINT_PAGE_TITLE}</h1>
+					<div className={classes.remaining}>
+						{MINT_SUPPLY_TEXT}: {totalSupply === null ? '...' : totalSupply} / {MAX_SUPPLY}
```

Comment:
`Supply : 0/1000`

Looks like sold out, but there are 1000 left.

Maybe something like `{MAX_SUPPLY - totalSupply} / {MAX_SUPPLY}` ? or change text to something like `SOLD: `

Reply:
I'm happy with changing the text to `SOLD: `. Most projects want the counter going up